crypto/keyring/codec.go

```diff
@@ -11,11 +11,13 @@ func init() {
 }
 
 // RegisterLegacyAminoCodec registers concrete types and interfaces on the given codec.
+// TODO how to remove Info entirely?
+// rename to LegacyInfo
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
-	cdc.RegisterInterface((*Info)(nil), nil)
+	cdc.RegisterInterface((*LegacyInfo)(nil), nil)
```

Comment:
Nope, cause `func NewLegacyMultiInfo(name string, pub cryptotypes.PubKey) (LegacyInfo, error) {` is used in `client/keys` package. According to linter rule, `LegacyInfo` return type should be public since `NewLegacyMultiInfo` is public too.